lib/get-config.js

```diff
@@ -78,13 +207,17 @@ function validateRules(config, options) {
   }
 }
 
+
+
 module.exports = function(options) {
   var config = options.config || readConfigFromDisk(options);
 
   ensureRootProperties(config);
   migrateRulesFromRoot(config, options);
 
-  config = processExtends(config);
+  config = processPlugins(config, options);
```

Comment:
Since we are mutating the `config` object in these functions, we shouldn't have to return and keep reassigning, right?